src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp

```diff
@@ -402,7 +402,14 @@ namespace OpenMS
     pickExperiment(input, output, boundaries_spec, boundaries_chrom, check_spectrum_type);
   }
 
-  void PeakPickerHiRes::pickExperiment(const PeakMap& input, PeakMap& output, 
+  struct SpectraPickInfo
+  {
+    uint32_t picked{0}; //< number of picked spectra
+    uint32_t total{0}; //< overall number of spectra
```

Comment:
```suggestion
    uint32_t total{0}; ///< overall number of spectra
```

Reply:
good catch... I always get this wrong...